test/api.js

```diff
@@ -899,42 +951,115 @@ describe('/v1', function() {
 
       describe('POST /:id', function() {
         var id = unique.id();
-        it('should update the client', function() {
-          var secret = unique.secret();
-          var imageUri = 'https://example.foo.domain/logo.png';
+
+        it('should forbid update to unknown developers', function() {
+          var id = unique.id();
+          var uid = unique(16).toString('hex');
+          var vemail = unique(4).toString('hex') + '@mozilla.com';
```

Comment:
I think I'm gonna create helper methods for user / tokens and client creation. There is just too much repetition in these tests